File: app/src/main/java/com/example/roomdatabase/MyDao.java
package com.example.roomdatabase;

import androidx.room.Dao;
import androidx.room.Delete;
import androidx.room.Insert;
import androidx.room.Query;
import androidx.room.Update;

import java.util.List;

@Dao
public interface MyDao {

    @Insert
    public void addEmployee(Employee employee);

    @Query("select * from Employee")
    public List<Employee> getEmployee();

    @Delete
    public  void deleteUser(Employee employee);

    @Update
    public  void updateEmployee(Employee employee);

}

File: app/src/main/java/com/example/jetpackexample/Database/MyDao.kt
package com.example.jetpackexample.Database

import androidx.lifecycle.LiveData
import androidx.room.Dao
import androidx.room.Insert
import androidx.room.Query
import com.example.jetpackexample.DataItems


@Dao
public interface MyDao{
    @Insert
    public fun addUser(user: DataItems)

    @Query("Select * from User")
    public fun getUser(): LiveData<List<DataItems>>

}